libdrizzle/pack.cc

```diff
@@ -481,3 +481,27 @@ static drizzle_return_t _pack_scramble_hash(drizzle_st *con,
 
   return DRIZZLE_RETURN_OK;
 }
+
+bool drizzle_check_unpack_error(drizzle_st *con)
+{
+  if (con->buffer_ptr[0] != 255)
```

Comment:
Not ATM. The values `(0, 254, 255)` are used as flags in various places but the meaning differs with the context. **PACKET_FLAG_0xFF** could work but doesn't connote the relation to the Err_packet